style.css

```diff
@@ -535,3 +545,314 @@ input {
   background: white;
   color: #5e6c84;
 }
+
+@media only screen and (min-width: 992px) {
+  body {
+    background: #e5e5e5;
+    max-width: 1100px;
+  }
+
+  .navbar {
+    width: 1350px;
```

Comment:
- [ ] I know that you are following the template by using the same width but for this project as we are adapting it to different kinds of screens. To mean it would be better for all with of containers to take `100%` instead of a fixed with. Kindly use `width: 100%` instead of `width: 1350px;` in the `navbar` class.This will adapt our design to all screens greater than 992px